### ISSUE TITLE
Error in dsm.cor(mod1, max.lag = 10) :    No column called Segment.Label in data

### ISSUE THREAD

**Issue by stewartford** [dill/mexico-data#1]
@dill 

When I try to run the following command at Line 205

```
dsm.cor(mod1,max.lag=10)
```

I get the following error:

Error in dsm.cor(mod1, max.lag = 10) : 
  No column called Segment.Label in data

**Comment by dill** [comment]
Hi Stewart,

A couple of things here:
1. `dsm.cor` require that you supply the name of the column that has the segment (sample labels), adapting your code to `dsm.cor(dsm.xy, max.lag=10, Segment.Label="Sample.Label")` should make this work.
2. There is an updated version of this vignette at http://distancesampling.org/R/vignettes/mexico-analysis.html, the `.Rmd` in this repository is the source of that. This includes the corrected `dsm.cor` call. Do make sure you have the latest versions of `dsm`, `mrds` and `Distance` installed from CRAN to make sure it runs okay.
3. In practise you might find that you need to subset the correlogram in some way to see if there is autocorrelation present in the data. It's possible when you have large numbers of sightings in diverse conditions that this just "averages out" in the correlogram, so you don't see what's going on. Subsetting by survey occasion, survey type or some other temporallly-based covariate may help.

Hope this helps!

**Comment by stewartford** [comment]
Thanks a lot Dave, I'll check out the updated vignette at distancsampling.org.  

I got here following the links in Appendix A of your 2013 paper and thought I'd try to follow the coding in the appendix while reading the paper. I've subsequently discovered that the mexico-analysis.Rmd shown online here is quite different to the one in the .zip I downloaded (which is closer to Appendix A of your paper), so the comment I made above might not make much sense here. 

Thanks again for your help!
